download_repo.sh

```diff
@@ -318,7 +318,7 @@ for arch in ${_arg_arch[@]}; do
         # fedorapeople
         # This will pull down the full contents, so we can update
         # the repository as necessary
-        override_uri="rsync://fedorapeople.org/project/modularity/repos/fedora/gencore-override/${version_path}/$arch"
+        override_uri="rsync://fedorapeople.org/project/modularity/repos/fedora/gencore-override/${version_path}/archful/$arch"
```

Comment:
You were right that there's a bug here, but 'archful' is the wrong path. You need to replace it with 'override' instead.